tools/bot/pkg/util/util.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package util
+
+import (
+	"bytes"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/iotexproject/go-pkgs/crypto"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/pkg/errors"
+)
+
+// GetPrivateKey get privatekey from keystore
+func GetPrivateKey(walletPath, addr, password string) (crypto.PrivateKey, error) {
```

Comment:
Can we use private key directly, which will be simpler for our deployment

Reply:
sure